examples/case-study-1/lib/Person.pm6

```diff
@@ -0,0 +1,31 @@
+use Red;
+
+#`{{{
+        create table person_tbl (
+                id    integer serial,
+                key   varchar(255) unique not null,
+                last  varchar(255) not null,
+                first varchar(255) not null
+                notes varchar(255)
+        );
+}}}
+
+unit model Person is rw;
+
+# this id (primary key) is used to create unique registration numbers
+# for various lists for easy use by humans
+has UInt $.id       is serial;
+
+# note the following is for user-defined keys to maintain unique
+# person status by personal attributes
+has Str  $.key      is column{:unique};
+
+# data:
+has Str  $.last     is column;
+has Str  $.first    is column;
+has Str  $.notes    is column{:nullable};
+
+# relations
+has      @.attends  is relationship{ .person-id, :model<Attend>  };
+has      @.emails   is relationship{ .person-id, :model<Email>   };
+has      @.presents is relationship{ .person-id, :model<Present> };
```

Comment:
It should be on of these:
- `is relationship(*.person-id, :model<Present>)`
- `is relationship(:column<person-id>, :model<Present>)`
- `is relationship{:column<person-id>, :model<Present>}`